mujoco_warp/_src/smooth.py

```diff
@@ -615,6 +615,11 @@ def qfrc_bias(m: Model, d: Data):
   wp.launch(qfrc_bias, dim=[d.nworld, m.nv], inputs=[m, d])
 
 
+def rne_postconstraint(m: Model, d: Data):
```

Comment:
this is old right - do you need to bring this up to date with HEAD?

Reply:
updated